### DISCUSSION TITLE
complex-valued data methods

### DISCUSSION THREAD

**Discussion by nschloe** [data-apis/consortium-feedback#9] [General] [answered]
I've looked at the upcoming standard for a little bit now, and played around with it in some of my libraries. One thing I couldn't find in the API are routines for handling complex-valued data, e.g.,
```python
xp.real(z)  # real part
xp.imag(z)  # imaginary part
xp.conj(z)  # complex conjugate
```
Is this planned to be added, or is there another way of handling it?

**Comment by leofang** [comment, marked as answer]
See https://github.com/data-apis/array-api/issues/373.
